src/tree/configurations/IngressItem.ts

```diff
@@ -7,17 +7,20 @@ import { ContainerApp, Ingress } from "@azure/arm-appcontainers";
 import { createGenericElement } from "@microsoft/vscode-azext-utils";
 import { AzureSubscription, ViewPropertiesModel } from "@microsoft/vscode-azureresources-api";
 import { ThemeIcon, TreeItem, TreeItemCollapsibleState } from "vscode";
-import { IngressConstants, azResourceContextValue } from "../../constants";
+import { IngressConstants } from "../../constants";
 import { localize } from "../../utils/localize";
 import { treeUtils } from "../../utils/treeUtils";
 import { ContainerAppModel } from "../ContainerAppItem";
 import { ContainerAppsItem, TreeElementBase } from "../ContainerAppsBranchDataProvider";
 
 const label: string = localize('ingress', 'Ingress');
 
-export class IngressItem implements ContainerAppsItem {
-    static contextValue: string = 'ingress|enabled';
-    readonly contextValue: string = `${IngressItem.contextValue}|${azResourceContextValue}`;
+const targetPortItemContextValue: string = 'targetPortItem';
+const visibilityItemContextValue: string = 'visibilityItem';
+
+export class IngressEnabledItem implements ContainerAppsItem {
```

Comment:
Since this class is renamed, consider also renaming the file to match

Reply:
I'm considering keeping the file name generic as just `IngressItem` here because it houses both the enabled and disabled versions of the tree item.  Just wanted it to match the same naming convention we were using for the Dapr tree items.

Reply:
Fair enough